asciidoc/courses/genai-mcp-build-custom-tools-python/modules/1-getting-started/lessons/2-setup/lesson.adoc

```diff
@@ -0,0 +1,151 @@
+= Setting Up Your Development Environment
```

Comment:
Should this lesson come first? I had already gone through the process of creating a virtualenv and install the mcp package.

The the learner clones the repo first, there could also be an example they can run / look at which shows the code the review in the mcp-python-sdk lesson.